SwiftStyleGuide.swift

```diff
@@ -135,3 +135,19 @@ private extension String {
     static let neptune = "neptune"
 
 }
+
+// Use one line per let in a guard with multiple lets.
+let maybe🌮: String? = "🥗"
+let maybe🥤: String? = "☕️"
```

Comment:
perhaps we shouldn't include the explicit types here, since it could be misleading, we typically don't set explicit types unless needed. 
Or maybe change these to a `var` so the optionality in the string makes more sense?

Reply:
Updating!